.github/workflows/short_unittest.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name:  Short Unit Tests
+
+on:
+  push:
+    branches-ignore: # master and develop run full test by other workflow
+      - master
+      - develop
+
+jobs:
+  build_and_unittest:
+    # this prevents the action to run on pull requests
+    if: github.event_name == 'push' && github.ref != 'refs/pull/*/merge'
```

Comment:
This is not working

Reply:
This is an option: https://github.com/marketplace/actions/skip-duplicate-actions

Reply:
The reason why the 2nd action did not run (was skipped) is because it had run before, not because the first action has run.

We can see this [here](https://github.com/aergoio/aergo/actions/runs/5077771284/jobs/9121449880?pr=207) where it states:

```
Skip execution because the exact same files have been successfully checked in run https://github.com/aergoio/aergo/actions/runs/5077707153
```

But that run is from the same action

Reply:
The other option bellow (joining them into a single job) may be better, and simpler

Reply:
I will try to implement and check the result